src/families/crypto_org/js-synchronisation.js

```diff
@@ -0,0 +1,52 @@
+//@flow
+import type { Account } from "../../types";
+import type { GetAccountShape } from "../../bridge/jsHelpers";
+import { makeSync, makeScanAccounts, mergeOps } from "../../bridge/jsHelpers";
+
+import { getAccount, getOperations } from "./api";
+
+const getAccountShape: GetAccountShape = async (info) => {
+  const { id, address, initialAccount } = info;
+  const oldOperations = initialAccount?.operations || [];
+
+  // Needed for incremental synchronisation
+  const startAt = oldOperations.length
+    ? (oldOperations[0].blockHeight || 0) + 1
+    : 0;
+
+  // get the current account balance state depending your api implementation
+  const {
+    blockHeight,
+    balance,
+    bondedBalance,
+    redelegatingBalance,
+    unbondingBalance,
+    commissions,
+  } = await getAccount(address);
+
+  // Merge new operations with the previously synced ones
+  const newOperations = await getOperations(id, address, startAt);
```

Comment:
to support pagination we would need to loop here. a bit like done here. 

https://github.com/LedgerHQ/ledger-live-common/blob/master/src/families/ethereum/synchronisation.js#L393-L408

Reply:
thanks, I added a loop.
(I also notice that the startAt takes the blockHeight, which does not work in my case because its just an incremental number. Not sure if its possible to store the "last index" fetch in the operation object to reduce the number of call)

Reply:
i'm not exactly sure what you mean. We don't enforce anything here, startAt is something you have internally in your implementation here, so you can do whatever you want, it's not generalised yet at this stage, the idea is just to "fetch all operations", in an incremental way (e.g. if user come with an account already filled, it will just ask the last page).

Reply:
Yeah I mean, I think it would be more optimal to store the index 'startA' in the Account so that you don't need to fetch all the pages every time. 
In the implementation in the example, they use the field "blockHeight" from the account before the 'startA' takes the blockHeight as a value instead of index. 

In my case I dont know if there is any field in Account where I can store that. It is not a big deal though, we can ignore this issue.

Reply:
makes sense. let's consider that in future iterations indeed.